.gitignore

```diff
@@ -14,3 +14,10 @@
 /denoDist
 /npm
 /deno
+/build
```

Comment:
is it docusaurus folder, if so can we name it more descriptive?

Reply:
yes.

Reply:
Don't think there is a way for us to customize this yet (at least I couldn't search anything on docs https://docusaurus.io/docs/next/installation#build